core/threshold/src/networking/tls.rs

```diff
@@ -96,8 +98,24 @@ pub struct AttestedVerifier {
     ignore_aws_ca_chain: bool,
 }
 
+/// We have to manually implement Debug for `AttestedVerifier` because Debug
+/// can't be derived for `user_data_verifier`.
+impl std::fmt::Debug for AttestedVerifier {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let mut f = f.debug_struct("AttestedVerifier");
+        let f = f
+            .field("root_hint_subjects", &self.root_hint_subjects)
+            .field("supported_algs", &self.supported_algs)
+            .field("pcr8_expected", &self.pcr8_expected);
```

Comment:
maybe we could add a field for `user_data_verifier`? maybe just to say if it's optional or not. also `ignore_aws_ca_chain`